packages/office-ui-fabric-react/src/components/Button/ActionButton/ActionButton.tsx

```diff
@@ -1,27 +1,11 @@
-import * as React from 'react';
-import { BaseButton } from '../BaseButton';
-import { BaseComponent, customizable, nullRender } from '../../../Utilities';
-import { IButtonProps } from '../Button.types';
+import { styled, nullRender } from '../../../Utilities';
+import { SplitButtonBase, ISplitButtonBaseProps } from '../';
 import { getStyles } from './ActionButton.styles';
 
-@customizable('ActionButton', ['theme'])
-export class ActionButton extends BaseComponent<IButtonProps, {}> {
-
-  /**
-   * Tell BaseComponent to bypass resolution of componentRef.
-   */
-  protected _shouldUpdateComponentRef = false;
-
-  public render() {
-    let { styles, theme } = this.props;
-
-    return (
-      <BaseButton
-        { ...this.props }
-        variantClassName='ms-Button--action ms-Button--command'
-        styles={ getStyles(theme!, styles) }
-        onRenderDescription={ nullRender }
-      />
-    );
-  }
-}
+export const ActionButton = styled(
```

Comment:
Love how straightforward the variants became!

Reply:
yeah! It highlights how a variant is just one of the base buttons with a set of styles passed in. I'm finding that I make variants on the fly inside of components when I need a styled button.